composer.json

```diff
@@ -16,10 +16,10 @@
         "php":">=5.4.0",
         "enygma/modler": "1.*",
         "robmorgan/phinx": "*",
```

Comment:
This requirement should also be changed because unbound constraints are very bad. The current version of your library is not compatible with all future major versions of ``robmorgan/phinx`` as major versions can break BC (this is the reason to use major versions).
And you cannot wait until there is a BC break in the library to decide fixing your requirement to avoid it, as all existing releases would still suffer from that and you cannot alter them.

However, I have not changed the constraint here as I don't know your codebase and so I don't know which version of phinx are actually supported.